src/widget-core/WidgetBase.ts

```diff
@@ -52,15 +52,38 @@ function isDomMeta(meta: any): meta is Base {
 	return Boolean(meta.afterRender);
 }
 
-const IGNORE_LIST: (string | symbol)[] = ['constructor', 'render'];
+const IGNORE_LIST: (string | symbol)[] = ['render', ...Object.getOwnPropertyNames(Object.getPrototypeOf({}))];
+
+const autoBindCache = new Map<any, string[]>();
 
 function autoBind(instance: any) {
-	let keys: string[] = Object.getOwnPropertyNames(instance.constructor.prototype);
+	let prototype = instance.constructor.prototype;
+
+	let keys: string[] = [];
+
+	if (autoBindCache.has(prototype)) {
+		keys = autoBindCache.get(prototype) as string[];
+	} else {
+		let p = prototype;
```

Comment:
Don't need to do this, just use `prototype`?

Reply:
@rorticus Did you see this? ^

Reply:
Removed extra variable `p`!